static/js/marker.js

```diff
@@ -39,7 +39,15 @@ var MarkerView = Backbone.View.extend({
 
 		this.$el.width(400);
 		this.$el.find(".title").text(TYPES_MAP[this.model.get("title")]);
-		this.$el.find(".description").text(this.model.get("description"));
+		this.$el.find(".roadType").text(this.model.get("roadType"));
```

Comment:
Saving the whole description as text wasn't a good idea. That's one of the
reasons we're changing it :)

About localization, I suppose localization.py should actually be a js file,
or json...

On Tue, May 12, 2015, 16:09 omerxx <notifications@github.com> wrote:

> In static/js/marker.js
> <https://github.com/hasadna/anyway/pull/250#discussion_r30131148>:
>
> > @@ -39,7 +39,15 @@ var MarkerView = Backbone.View.extend({
> >
> >  		this.$el.width(400);
> >  		this.$el.find(".title").text(TYPES_MAP[this.model.get("title")]);
> > -		this.$el.find(".description").text(this.model.get("description"));
> > +		this.$el.find(".roadType").text(this.model.get("roadType"));
>
> I understand the Integers idea. Thought it should be kept as description
> was saved as text in the DB.
>
> I will first change python to pull Integers from the DB.
> Then have 'localize_value' function to do the same only in Marker.js
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hasadna/anyway/pull/250/files#r30131148>.
>